yarn.lock

```diff
@@ -10625,9 +10625,10 @@ rollup@1.31.0:
     "@types/node" "*"
     acorn "^7.1.0"
 
-rollup@endojs/endo#rollup-2.7.1-patch-1:
-  version "2.70.1-endo.1"
-  resolved "https://codeload.github.com/endojs/endo/tar.gz/54060e784a4dbe77b6692f17344f4d84a198530d"
+rollup@^2.77.4-1:
+  version "2.79.1"
```

Comment:
Did you mean to resolve to 2.79?  If not, maybe `rollup@~2.77.4` is what you meant?